docker/docker-compose.yml

```diff
@@ -1,22 +1,47 @@
-web:
+worker: &worker
   build: ../
-  command: python manage.py runserver --settings=kuma.settings.docker 0.0.0.0:8000
+  command: supervisord -n -c /app/docker/supervisor-celery.conf
   volumes:
     - ../:/app
-  ports:
-    - "8000:8000"
   links:
     - memcached
     - mysql
     - elasticsearch
     - redis
```

Comment:
The workers do yeah for db queries, search indexing, cached things. Redis is the broker so that's needed here too.